modules/genomics_nextflow/outputs.tf

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+output "project_id" {
+  description = "Project ID where resources where created"
+  value       = local.project.project_id
+}
+output "nextflow_server_instance_id" {
+  description = "VM instance name running the nextflow server"
+  value       = google_compute_instance.nextflow_server.name
+}
+output "nextflow_server_zone" {
+  description = "Google Cloud zone in which the server was provisioned"
+  value       = var.zone
+}
+
+output "nextflow_service_account_email" {
+  description = "Email address of service account running the server and worker nodes"
+  value       = module.nextflow_service_account.email
+}
+
+output "gcs_bucket_url" {
+  description = "Google Cloud Storage Bucket configured for workflow execution"
+  value       = google_storage_bucket.nextflow_workflow_bucket.url
+}
+
+output "gcloud_ssh_command" {
+  description = "To connect to the Nextflow instance using Identity Aware Proxy, run the following command"
+  value       = "gcloud compute ssh ${google_compute_instance.nextflow_server.name} --zone=${var.zone} --project ${local.project.project_id}"
```

Comment:
SSHing to the Nextflow Server (from my localhost) is erroring out. Can you please look into it..

Reply:
The VM was not created when I tried the code as the Shielded VM org policy was still enforced
it works fine for me now

Reply:
I've tested now twice after the latest commits and using a tfvars file, and I'm able to ssh to the VM

Reply:
@guptamukul-google I have added the Service account user permission to the trusted users which should fix the issue

Reply:
I just tested it in my own deployment of this module and can confirm that ssh issue is resolved.